sourced/ml/__main__.py

```diff
@@ -186,6 +187,22 @@ def add_parser(name, help_message):
         "input", help="Path to the model file, URL or UUID.")
     dump_parser.add_argument("--gcs", default=None, dest="gcs_bucket",
                              help="GCS bucket to use.")
+    # ------------------------------------------------------------------------
+    merge_df = add_parser("merge-df", "Merge DocumentFrequency models to a singe one.")
+    merge_df.set_defaults(handler=merge_df_entry)
+    add_filter_arg(merge_df)
+    add_min_docfreq(merge_df)
+    add_vocabulary_size_arg(merge_df)
+    merge_df.add_argument(
+        "-o", "--output", required=True,
+        help="Path to the merged document frequencies model.")
```

Comment:
DocumentFrequencies

We should use exactly the same class name everywhere to reduce confusion